modules/@apostrophecms/util/ui/public/video.js

```diff
@@ -1,62 +1,75 @@
-apos.util.widgetPlayers['@apostrophecms/video'] = function(el, data, options) {
+apos.util.widgetPlayers['@apostrophecms/video'] = {
+  selector: '[data-apos-video-widget]',
+  player: function(el) {
+    const videoUrl = el.getAttribute('data-apos-video-url');
 
-  queryAndPlay(
-    el.querySelector('[data-apos-video-player]'),
-    apos.util.assign(data.video, {
+    if (!videoUrl || el.getAttribute('data-apos-widget-played')) {
+      return;
+    }
+
+    el.setAttribute('data-apos-widget-played', true);
+
+    queryAndPlay(el, {
+      url: videoUrl,
       neverOpenGraph: 1
```

Comment:
We should dump our open graph thing, which we never really used effectively in 2.x and just got headaches from. But, that's a ticket for later.

Reply:
I added a ticket for this.